.vscode/launch.json

```diff
@@ -22,6 +22,7 @@
             "preLaunchTask": "buildproject",
             "runtimeExecutable": "${execPath}",
             "args": [
+                "--disable-extensions",
```

Comment:
This was handled in PR #160 and quite a bit more. Make sure you update to the latest before these test additions get added.